syntax = "proto3";

package octo;

// Definition des Dienstes f체r Nachrichten
service MessageService {
  // Sendet eine Nachricht
  rpc OctoMessage (OctoRequest) returns (OctoResponse) {}

  // Ruft das Datenformat ab
  rpc GetDataFormat (GetDataRequest) returns (OctoResponse) {}
}

// Die Anforderungsnachricht enth채lt die JSON-Nachricht
message OctoRequest {
  string json_message = 1;
}

// Die Antwortnachricht enth채lt die JSON-Nachricht
message OctoResponse {
  string json_message = 1;
  string test = 2;
}

// Die Anforderungsnachricht f체r das Datenformat
message GetDataRequest {}

